app/database/migrations/2014_12_31_230032_CreateComponentGroupsTable.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateComponentGroupsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('component_groups', function (Blueprint $table) {
```

Comment:
You've reference the actual instance there, but you want to call the "facade". It's basically a proxy to the container.